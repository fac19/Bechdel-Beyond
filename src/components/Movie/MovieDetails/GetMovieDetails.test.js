import React from 'react';
import renderer from 'react-test-renderer';

import MovieDetailsPage from './GetMovieDetails';

test('Check movie page tests are running', () => {
	expect(true).toBeTruthy();
});

it('Get movie details section renders correctly on movie page', () => {
	const movieDetails = renderer.create(<MovieDetailsPage />).toJSON();
	expect(movieDetails).toMatchSnapshot();
});
