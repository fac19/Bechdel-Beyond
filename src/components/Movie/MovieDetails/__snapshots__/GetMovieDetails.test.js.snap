// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get movie details section renders correctly on movie page 1`] = `
<h1>
  Locading
</h1>
`;
